### ISSUE TITLE
Consider sending Accept-Language to customize server error message

### ISSUE THREAD

**Issue by joelverhagen** [NuGet/Home#5046]
Today, all of the error messages returned by NuGetGallery to the client are in English.

The client (nuget.exe and Visual Studio) knows the user's locale (this is something .NET can tell us). Client can send this information to the server with the `Accept-Language` header and the server can react by giving localize error messages.

Today, most of the client error experience when contacting the server is through the response code and reason phrase (e.g. `409 Conflict`). To make use of the `Accept-Language` header we would need to formalize the response body and put the error messages in there. Reason phrases do not support UTF-8 as far as I know.

In short, two changes should be made to provide localized error messages:

1. Client sends `Accept-Language` header.
1. Server returns localized error message in the response body, which the client displays.
